ajax/libs/preact-compat/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "preact-compat",
+  "version": "3.15.0",
```

Comment:
We don't need version field if adding lib via single package.json. Please remove it. Thanks.